src/main/java/com/learning/libraryManagement/service/BooksService.java

```diff
@@ -1,4 +1,27 @@
 package com.learning.libraryManagement.service;
 
+
+import com.learning.libraryManagement.model.Books;
+import com.learning.libraryManagement.repository.BooksRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
+
 public class BooksService {
+    @Autowired
+    BooksRepository booksRepository;
+    public List<Books> getBooks(List<Books> books){
+
+       this.booksRepository.saveAll(books);
```

Comment:
when you are trying to get you should use find all

Reply:
Initial I done it for create books details so i use saveAll .Now added findAll and SaveAll for get and Post